### ISSUE TITLE
It is possible to open multiple tabs inside the app with the same content

### ISSUE THREAD

**Issue by SebastianOsinski** [TechxArtisanStudio/Openterface_MacOS#3]
As shown on the video, it is possible to enable tab bar in the app by going to `View -> Show Tab Bar`. After that it is possible to open multiple tab bars which all show the same content. It is even possible to drag the tab bar out of the window and create new window of Openterface. 

If this is intended to be a feature (to support multiple Openterface devices at once) then it should not be possible to have more than one tab connected to the same device.

https://github.com/TechxArtisanStudio/Openterface_MacOS/assets/2934986/61c1512c-ee58-4607-b00b-f47a9c37a828

**Comment by youyoubilly** [comment]
Yes, that's exactly what we're going for — being able to use multiple mini-KVMs at the same time, and tabs make it easy to switch between them. We're still doing some basic testing on the USB tree, just to make sure multiple mini-KVMs matching K/M with video source correctly.
![mmexport1715144459702.jpg](https://github.com/TechxArtisanStudio/Openterface_MacOS/assets/54311293/3d0c65b1-0c7c-4019-aef4-d85f8fc75638)
